src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1180,10 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
-     *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
-     */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    * Removes prefix from the given fullString if prefix occurs at the start of the string.
+    */
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replaceFirst(prefix, "");
```

Comment:
Does this method replace prefix if it occurs at the beginning of fullString or the first occurance of prefix in anywhere in fullString. 

Can be improved to make it more abstract.